integration/apps/rails-five/app/controllers/health_controller.rb

```diff
@@ -16,7 +16,6 @@ def detailed_check
       profiler_threads: Thread.list.map(&:name).select { |it| it && it.include?('Profiling') },
       telemetry_enabled: Datadog.configuration.telemetry.enabled,
       telemetry_client_enabled: Datadog.send(:components).telemetry.enabled,
-      telemetry_worker_enabled: Datadog.send(:components).telemetry.worker.enabled?
```

Comment:
The internal worker instance is now private, even it's an implementation concern.